cadc-vos-server/src/main/java/ca/nrc/cadc/vos/server/web/restlet/action/GetNodeAction.java

```diff
@@ -202,7 +244,15 @@ public NodeActionResult performNodeAction(Node clientNode, Node serverNode)
             else
             {
                 // get as many children as allowed
-                nodePersistence.getChildren(cn, resolveMetadata);
+                if (sortParamURI == null && sortAsc == null) {
```

Comment:
This whole duplicate block could be avoided by initializing pageLimit to null, getting rid of "paginate", and just using the previous block. It's more obvious with excess logic being duplicated.